src/main/resources/petriNets/leukemia_en.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <document xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:noNamespaceSchemaLocation='https://petriflow.com/petriflow.schema.xsd'>
+    <id>leukemia/leu</id>
```

Comment:
why does an english version of this net exist when it could be internationalised in one process?

Reply:
The process is for testing purposes only. The original purpose of 'en version' is not known to me.